.github/workflows/ci.yml

```diff
@@ -441,10 +441,12 @@ jobs:
         shell: bash
         run: |
           set -ex
-          cargo test --profile ci-rust --workspace ${{ env.cargo-exclude-unused-crates }}
+          cargo test --profile ci-rust --workspace ${{ env.cargo-exclude-unused-crates }} -- --nocapture
```

Comment:
There is a way to only show the output log for failing tests only (included the tests that time out) ?

Reply:
I don't find

Reply:
The question can be rewritten as:

What append when a test fail without using `--nocapture` ?

Reply:
It logs nothing in case of timeout